setup-elytron-guides/setup_elytron_linux.adoc

```diff
@@ -0,0 +1,163 @@
+. It is easiest to install Git on Linux using the preferred package manager of your Linux distribution. Here are some examples of how to install Git on common Linux distributions:
+    .. Ubuntu: `sudo apt-get install git`
+    .. Fedora: `sudo dnf install git`
+    .. Arch: `sudo pacman -S git`
+. Now run `git --version` again. You should now successfully see the Git version that was installed.
+. If you already had any IDEs open, close and reopen them now to load your changes.
+
+== Cloning WildFly Elytron
+
+. First, configure Git so that it can attribute your commits to you. Run the following commands in the terminal, substituting your own name and email:
++
+[source,bash]
+----
+git config --global user.email "<you@example.com>"
+git config --global user.name "<Your Name>"
+----
+
+. Next, you'll need a GitHub account. If you don't already have one, sign up for one at https://github.com/[github.com]. Otherwise, make sure you're logged in.
+. Fork the https://github.com/wildfly-security/wildfly-elytron[WildFly Elytron project] into your GitHub account by clicking on the "Fork" button at the top right.
++
+[.responsive-img_capped_50-width.text-center]
+image::setup-elytron_fork-repo.png['Screenshot of buttons at top of wildfly-elytron repo, including "Fork" button']
+
+. Now go to your fork of the repository, the URL will be *https://github.com//wildfly-elytron*. Replace `<USERNAME>` with your GitHub username.
+. At the top right, you'll see there is a green "Code" button. Click on that, and for the easiest setup choose the "HTTPS" tab. Click on the button next to the URL to copy it.
++
+[.responsive-img_capped.text-center]
+image::setup-elytron_github-clone.png['Screenshot of options under "Code" button, under HTTPS tab.Default URL is https://github.com//wildfly-elytron.git']
+
+. Open your terminal and navigate to the directory where you want to clone this project. Then enter `git clone [URL]` and replace `[URL]` with the URL you copied in step 4. Now you should see a directory called `wildfly-elytron` that contains the code for the WildFly Elytron project.
+. Next, add a remote reference to upstream, for pulling future updates from the source repository. Execute the following command in the terminal:
++
+[source,bash]
+----
+cd wildfly-elytron
+git remote add upstream https://github.com/wildfly-security/wildfly-elytron.git
+----
+
+== Building WildFly Elytron
+
+. Open your terminal.
+. Navigate to the wildfly-elytron project directory and run `mvn clean install`. This will build the project.
+. Check out the link:../../getting-started-for-developers/[*Getting Started for Developers guide*] to learn more about the WildFly Elytron project and how to run tests.
+
+== _[OPTIONAL] Temporary Configuration_
+
+If you want to undo the changes to the default Java and Maven versions, you can remove the modifications after you finish. We will not uninstall the programs, in case you want to use them later, but this can be done as follows:
```

Comment:
Just minor, but this sentence should be updated since it says we won't uninstall but then shows how to uninstall :)

Reply:
completely intentional, not a missed section at all :)